apps/emqx_conf/src/emqx_cluster_rpc.erl

```diff
@@ -262,6 +262,8 @@ fast_forward_to_commit(Node, ToTnxId) ->
 
 %% @private
 init([Node, RetryMs]) ->
+    %% Workaround for https://github.com/emqx/mria/issues/94:
```

Comment:
For my own information: how does the line below relate to the flaky test issue?

Reply:
Both this issue and the flaky test are likely caused by shard rlookup not updated properly